src/unitxt/operators.py

```diff
@@ -1603,12 +1603,7 @@ def process(self, stream: Stream, stream_name: Optional[str] = None) -> Generato
             ), f"Operator {metric_name} must be a Metric"
 
             if not self.calc_confidence_intervals:
-                if isinstance(metric, MetricWithConfidenceInterval):
-                    metric.disable_confidence_interval_calculation()
-                elif isinstance(metric, MetricPipeline) and isinstance(
-                    metric.metric, MetricWithConfidenceInterval
-                ):
-                    metric.metric.disable_confidence_interval_calculation()
+                metric.disable_confidence_interval_calculation()
```

Comment:
do we also need an abstract `disable_confidence_interval_calculation()` in `Metric`?

Reply:
currently IIUC only the "overrides" were defined ..